auth0/resource_auth0_prompt_custom_text.go

```diff
@@ -153,13 +157,13 @@ func getPromptAndLanguage(d *schema.ResourceData) (string, string, error) {
 func marshalCustomTextBody(b map[string]interface{}) (string, error) {
 	bodyBytes, err := json.Marshal(b)
 	if err != nil {
-		return "", fmt.Errorf("Failed to serialize the custom texts to JSON: %w", err)
+		return "", fmt.Errorf("failed to serialize the custom texts to JSON: %w", err)
 	}
 
 	var buffer bytes.Buffer
 	const jsonIndentation = "    "
 	if err := json.Indent(&buffer, bodyBytes, "", jsonIndentation); err != nil {
-		return "", fmt.Errorf("Failed to format the custom texts JSON: %w", err)
```

Comment:
**Question (non-blocking):** Is there an automated process that prompts these changes?

Reply:
You mean if these are covered by tests? 

Some are and some aren't. The tests mostly check for the existence of an error but not if the error starts capitalized or not. In Go errors should not be capitalized or end with a punctuation.